src/components/dashboard/dashboard.service.ts

```diff
@@ -49,6 +50,14 @@ export class DashboardService implements OnDestroy {
     private _onDestroy = new Subject<void>();
 
     constructor() {
+        combineLatest(this.layout$, this.widgets$, this.dimensions$, this.stacked$)
+        .pipe(tick(), filter(([layout, widgets, dimensions, stacked]) => stacked === false), takeUntil(this._onDestroy))
+        .subscribe(([layout, widgets, dimensions, stacked]) => {
```

Comment:
Can you indent `pipe` and `subscribe` to improve readability here?

Reply:
Yeah no worries